packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -811,3 +846,28 @@ class ScriptCache {
     _inProgress.clear();
   }
 }
+
+/// A dev time class to help trace DevTools application events.
+class TimingLogger {
```

Comment:
let's move this to utils since it is not specific to the debugger. Also, maybe rename to DebugTimingLogger to make it clear that this a is a debugging class

Reply:
good idea, will do